test/trackler/problem_test.rb

```diff
@@ -4,25 +4,65 @@
 class ProblemTest < Minitest::Test
   def test_existing_problem
     problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
-
     assert problem.exists?, "hello world files not found in the metadata dir"
+  end
+
+  def test_slug
```

Comment:
If we have a table then it will stop evaluating tests on the first failure, which is unfortunate. Unless we do metaprogramming to create separate methods, which just seems like too much indirection for tests.

All in all I agree that it's not worth it here to DRY out the tests.